packages/clay-core/docs/focus-trap.js

```diff
@@ -11,12 +11,16 @@ const exampleImports = `import {Button, FocusTrap} from '@clayui/core';
 import React, {useState, useRef} from 'react';`;
 
 const exampleCode = `const FocusTrapTrigger = () => {
-    const [active, setActive] = useState(false);
+	const [active, setActive] = useState(false);
 	const activateButtonRef = useRef(null);
 
 	useEffect(() => {
 		if (active) {
-			return () => activateButtonRef.current?.focus();
+			return () => {
+				if (activateButtonRef.current) {
+					activateButtonRef.current.focus()
+				}
+			};
```

Comment:
Hmm that's strange the previous implementation is also correct.

Reply:
@matuzalemsteles idk node? was having trouble with the optional chaining operator. Maybe we are not running these files through babel?

Reply:
Probably yes... strange that it didn't break before. Well but we can go with this to avoid headache now.